default/index.js

```diff
@@ -59,18 +64,43 @@ function readDirectory(dirName, stream) {
   return files;
 }
 
-// If stream is true, a stream is provided instead of a buffer for
-// each file object along with the corresponding size
-function getDefaultProjects(stream) {
-  var DefaultProjects = {};
+function cacheProjectFiles() {
```

Comment:
I see, this is run when the module is required in but `getDefaultProject` is run when the API is called.
Can you move this function below the `getDefaultProject` one so that when I see the first statement that is not a function, I don't have to scroll up past a function (viz. `getDefaultProject`) that is only referenced later on.